manager/integration/tests/test_rwx.py

```diff
@@ -89,11 +544,63 @@ def test_rwx_parallel_writing():
     10. Compute md5sum.
     11. Check the data md5sum in longhorn-nfs-provisioner volume
     """
-    pass
+    nfs_provisioner_pod = k8sclient.CoreV1Api().list_pod_for_all_namespaces(
+        label_selector="app=longhorn-nfs-provisioner", watch=False)
+    nfs_provisioner_pod_name = nfs_provisioner_pod.items[0].metadata.name
 
+    statefulset_name = 'statefulset-parallel-write-test'
+    statefulset['metadata']['name'] = \
+        statefulset['spec']['selector']['matchLabels']['app'] = \
+        statefulset['spec']['serviceName'] = \
+        statefulset['spec']['template']['metadata']['labels']['app'] = \
+        statefulset_name
+    statefulset['spec']['volumeClaimTemplates'][0]['spec']['storageClassName']\
+        = NFS_STORAGECLASS_NAME
+    statefulset['spec']['replicas'] = 1
 
-@pytest.mark.skip(reason="TODO")
-def test_rwx_statefulset_scale_down_up():
+    create_and_wait_statefulset(statefulset)
+
+    statefuleset_pod_name = statefulset_name + '-0'
+    statefuleset_pvc_name = \
+        statefulset['spec']['volumeClaimTemplates'][0]['metadata']['name'] \
+        + '-' + statefulset_name + '-0'
+    statefulset_volume_name = get_volume_name(core_api, statefuleset_pvc_name)
+
+    pvc_name = 'pvc-parallel-write-test'
+    pvc['metadata']['name'] = pvc_name
+    pvc['spec']['storageClassName'] = NFS_STORAGECLASS_NAME
+    core_api.create_namespaced_persistent_volume_claim(
+        body=pvc, namespace='default')
+
+    pod_name = 'pod-parallel-write-test'
+    pod['metadata']['name'] = pod_name
+    pod['spec']['volumes'] = [create_pvc_spec(pvc_name)]
+    create_and_wait_pod(core_api, pod)
+    pod_volume_name = get_volume_name(core_api, pvc_name)
+
+    with Pool(2) as p:
+        p.map(write_data_into_pod, [pod_name, statefuleset_pod_name])
+
```

Comment:
I think you need to wait for completion of the above parallel tasks?

Reply:
Only after the parallel write is completed, it goes to the next line to execute.